tlatools/org.lamport.tlatools/src/pcal/ParseAlgorithm.java

```diff
@@ -1440,7 +1442,7 @@ public static AST.SingleAssign GetSingleAssign() throws ParseAlgorithmException
          * PeekAtAlgToken(1), so LAT[0] contains the next token.           *
          ******************************************************************/
        result.lhs = GetLhs() ;
-       GobbleThis(":=") ; 
+       GobbleThis(":=", "≔") ; 
```

Comment:
> How is ≔ rendered in relation to the other symbols in the different TLA+ IDEs?

Well, here's what it looks like in Emacs

![image](https://github.com/tlaplus/tlaplus/assets/7935057/40a612c4-9cfe-43fc-b722-bf0276f9b9c9)

> For me here on Github, ≔ only becomes discernible at zoom levels above 150%.

I think it depends. I can clearly see right here in your comment the `≔` symbol without having to zoom in.

As a matter of fact, my native language has a symbol such as `ё` which you'd probably consider even worse. But I assure you we have no problem with that symbol 😊

Reply:
![image](https://github.com/tlaplus/tlaplus/assets/88777/8540d056-e074-40bd-9f3a-ca512221594e)

Reply:
It's good to know that Emacs uses a sensible symbol. However, since most TLA+ users use the Toolbox or the VSCode extension, this is where we need to ensure it renders correctly out of the box.

Reply:
>  since most TLA+ users use the Toolbox or the VSCode extension

Yeah, I confirm the character looks terrible in VScode and [reordering fonts to render emoji with a different one](https://github.com/microsoft/vscode/issues/32840#issuecomment-345923580) simply doesn't work for some reason.

Anyway, it is a font problem.

> this is where we need to ensure it renders correctly/decently out of the box

It's impossible. I mean, tla+toolbox kind of can try to set up fonts in such a way so this emoji renders correctly, but in the end you can't guarantee the font won't be missing on the end-user's system. Not to mention, for external editors such as VSCode you can't do anything at all.

Reply:
> > since most TLA+ users use the Toolbox or the VSCode extension
> 
> Yeah, I confirm the character looks terrible in VScode and [reordering fonts to render emoji with a different one](https://github.com/microsoft/vscode/issues/32840#issuecomment-345923580) simply doesn't work for some reason.
> 
> Anyway, it is a font problem.
> 
> > this is where we need to ensure it renders correctly/decently out of the box
> 
> It's impossible. I mean, tla+toolbox kind of can try to set up fonts in such a way so this emoji renders correctly, but in the end you can't guarantee the font won't be missing on the end-user's system. Not to mention, for external editors such as VSCode you can't do anything at all.

Primarily, it is *our* problem if the user experience is poor.  Declaring it impossible to improve is not acceptable.  If a good user experience cannot be provided out of the box, we need to at least provide documentation to guide users on how to fix it.

Reply:
After discussion in the monthly community meeting this morning I have opened a discussion in the google group about this: https://groups.google.com/g/tlaplus/c/P-GLlN07ARo/m/I5hGjR6AFAAJ